resources/manifests/kubelet-pod-checkpointer-psp-cluster-role-binding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: kubelet-pod-checkpointer-psp
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: pod-checkpointer-psp
+subjects:
+- kind: Group
+  name: system:nodes
```

Comment:
Not sure about this at all, but I think we can use `system:nodes:<node-name>` maybe? I think we need to enable Node admission controller: https://kubernetes.io/docs/reference/access-authn-authz/node/ (that is recocommended anyways, it seems). In that case, this might be out of scope for the PR (and not sure we can use it easily, because we need to know the name of *all* the masters, for HA setups).

Or isn't there some other group that is more specific to the controller in some way?

Reply:
Kubelet certificate has `system:nodes` group specified and user `kubelet`. Once we switch to Node authorization mode with Node restriction admission, then we will change this approach. See kinvolk/PROJECT-Lokomotive-Kubernetes#240.

I agree though, that this grant could be limited to controller nodes only, but I think this is way out of scope for now.

Reply:
Great, I agree is out of scope and if we already have an issue for this it seems great.

As far as I know, then, there is no better way until that is done to restrict this even more. Am I correct? If this is the case, consider this resolved :)

Reply:
I don't know a better way of doing it :)